test/integration/BUILD

```diff
@@ -795,3 +795,23 @@ envoy_cc_test(
         "//test/test_common:utility_lib",
     ],
 )
+
+envoy_cc_test(
+    name = "aws_metadata_fetcher_integration_test",
+    srcs = [
+        "aws_metadata_fetcher_integration_test.cc",
+    ],
+    tags = ["exclusive"],
```

Comment:
Why is this test tagged as exclusive? This acts as a serialization step and forces the total test run to increase by > 30s, as it can't be parallelized.

Reply:
@htuch I'm not sure why this is set this way but @ivitjuk is no longer working on this.

I think this is safe to remove. I can open a PR.

Reply:
Integration test is creating an instance of the Envoy which is acting as a metadata endpoint. If we remove exclusive, wouldn't we risk port collision with other integration tests that create listeners?

Reply:
Unless you are binding to fixed ports (which I didn't spot, we typically don't do this for the reason you state), this is safe to not be exclusive. The tests tend to bind to port zero, which allows safe concurrent execution.

Reply:
Thanks, @lavignes can you make a PR to remove `exclusive`?

Reply:
@ivitjuk @htuch done! #7545

Reply:
@lavignes thanks, appreciated!